examples/linear_model/plot_tweedie_regression_insurance_claims.py

```diff
@@ -0,0 +1,594 @@
+"""
+======================================
+Tweedie regression on insurance claims
+======================================
+
+This example illustrates the use of Poisson, Gamma and Tweedie regression on
+the French Motor Third-Party Liability Claims dataset, and is inspired by an R
+tutorial [1]_.
+
+Insurance claims data consist of the number of claims and the total claim
+amount. Often, the final goal is to predict the expected value, i.e. the mean,
+of the total claim amount per exposure unit also referred to as the pure
+premium.
```

Comment:
I assume exposure is a unit of time here, but it seems that in the insurance world, exposure refers to the risk associated with an individual.

I'm a bit confused, what does exposure mean in this context?

Reply:
Exposure in insurance usually means a measure of "volume of risk pro rata temporis". In this dataset, the exposure of one row is the duration of insurance coverage for one policy in unit of years, i.e. 1 = 1 year, 0.5041=184 days (e.g. from 1 July to 31 Dec).